core/src/main/java/io/grpc/util/MutableHandlerRegistry.java

```diff
@@ -79,6 +79,11 @@ public ServerServiceDefinition addService(BindableService bindableService) {
     return addService(bindableService.bindService());
   }
 
+  /**
+   * Remove a registered service
+   *
+   * @return true if the service was found and is removed.
```

Comment:
Remove "and is removed". That makes it confusing since it makes you wonder, "why wouldn't it be removed?" You can say "if the service was found to be removed"